packages/terra-form-select/tests/wdio/select-field-spec.js

```diff
@@ -52,7 +52,7 @@ Terra.describeViewports('Select Field', ['tiny'], () => {
     Terra.it.validatesElement();
 
     it('should open the dropdown by clicking the toggle', () => {
-      browser.click('#select-field');
+      browser.click('[class*="tag"]');
```

Comment:
These wildcard selectors can be flaky if we add different classes down the road that include the tag suffix. There are data attributes we may be able to use on the frame and toggle buttons.

Reply:
Resolved in 85bc519